src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -217,13 +219,14 @@ public void run(StorageService storage, Tuple<Path, Blob> tuple) throws Exceptio
     }
 
     @Override
-    Tuple<Path, Blob> parse(String... args) {
+    Tuple<Path, Blob> parse(String... args) throws IOException {
       if (args.length < 2 || args.length > 3) {
         throw new IllegalArgumentException();
       }
       Path path = Paths.get(args[0]);
+      String contentType = Files.probeContentType(path);
```

Comment:
The default probeContentType() used to be fairly incomplete on the types it actually recognized.
Have you verified this actually works as expected or is some fallback needed?